.gitignore

```diff
@@ -4,6 +4,7 @@
 # Python
 *.py[cod]
 __pycache__/
+venv/
```

Comment:
You could break this out into a different PR, probably not worth it, but if you wanted to this change is unrelated and does not require the same level of review.